packages/transmission-rpc/transmission-rpc.1.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "Benoit Bataille <benoit.bataille@gmail.com>"
+authors: [ "Benoit Bataille <benoit.bataille@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/bataille/ocaml-transmission-rpc"
+bug-reports: "https://github.com/bataille/ocaml-transmission-rpc/issues"
+dev-repo: "https://github.com/bataille/ocaml-transmission-rpc"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "transmission-rpc"]
+]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+depends: [
+  "cohttp"
+  "lwt"
+  "ocamlfind" {build}
+  "ppx_deriving"
+  "ppx_deriving_yojson"
```

Comment:
I believe this needs to be `{>= "2.0"}` due to your use of helpers introduced in that release: <https://github.com/whitequark/ppx_deriving_yojson/blob/master/CHANGELOG.md>.